src/cascadia/TerminalSettingsEditor/Launch.cpp

```diff
@@ -37,4 +37,20 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         const auto profile{ winrt::unbox_value<Model::Profile>(value) };
         _State.Settings().GlobalSettings().DefaultProfile(profile.Guid());
     }
+
+    Windows::Foundation::Collections::IObservableVector<IInspectable> Launch::DefaultTerminals()
+    {
+        Windows::Foundation::Collections::IObservableVector<IInspectable> vec;
+        return vec;
+    }
+
+    IInspectable Launch::CurrentDefaultTerminal()
+    {
+        return nullptr;
+    }
+
+    void Launch::CurrentDefaultTerminal(const IInspectable& value)
+    {
+        value;
+    }

```

Comment:
I know we talked about it offline, but is there already an issue to track hooking this up? Maybe add
```c++
// TODO GH#XXXX: Add DefTerm to Settings UI
```

Reply:
The funny thing is, all that's left here is just calling the Get/Set APIs you created, and populating that `DefaultTerminals` list in `OnNavigatedTo` haha. Really excited to get this working!!!!!! 🎉

Reply:
I'll leave the comment and finish it after this one. Glad to hear I'm mostly there!